docs/config/cluster.rst

```diff
@@ -1259,15 +1259,13 @@ The following settings can be used to configure the behavior of the
   | *Default:*   ``-1``
   | *Runtime:*  ``no``
 
-  The setting ``gateway.expected_nodes`` defines the number of nodes that
-  should be waited for until the cluster state is recovered immediately. The
-  value of the setting should be equal to the number of nodes in the cluster,
-  because you only want the cluster state to be recovered after all nodes are
-  started.
+  The setting ``gateway.expected_nodes`` defines the total number of nodes 
+  expected in the cluster. It is evaluated together with recover_after_nodes
+  to decide if the cluster can proceed with recovery.
```

Comment:
```suggestion
  to decide if the cluster can start with recovery.
```
Not sure, is any recovery happening already before reaching `gateway.expected_nodes`?